intake/cli/client/subcommands/drivers.py

```diff
@@ -0,0 +1,79 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2018, Anaconda, Inc. and Intake contributors
+# All rights reserved.
+#
+# The full license is in the LICENSE file, distributed with this software.
+#-----------------------------------------------------------------------------
+'''
+
+'''
+
+from __future__ import print_function
```

Comment:
is intake Py2 compatible ?

Reply:
It was initially, but we found it not worthwhile to maintain as other pydata projects were dropping support

Reply:
Should this line be removed then ?